Xamarin.Forms.Xaml/MarkupExtensions/FontImageExtension.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Xamarin.Forms.Xaml
+{
+	[AcceptEmptyServiceProvider]
+	[ContentProperty(nameof(Glyph))]
+	public class FontImageExtension : IMarkupExtension<ImageSource>
+	{
+		public string FontFamily { get; set; }
+		public string Glyph { get; set; }
+		public Color Color { get; set; }
+		public double Size { get; set; } = 30d;
```

Comment:
not sure about setting a default. does FontImageSource requires a Size ?

Reply:
Size already defaults to 30 on FontImageSource, but if you don't set the default here, it'll revert to `0`.

Perhaps you can use `FontImageSource.SizeProperty.DefaultValue`

Reply:
alternatively, I don't know if FontImageSource has a concept of a magic value `-1` to roll over to default.

Reply:
My thought process behind this is the default size for `FontImageSource` is `30d`. If someone uses this markup extension it will default to 0 unless they explicitly provide a size. 

Perhaps we add a static default in `FontImageSource` and that way both the markup extension and `FontImageSource` can reference the default size.

**FontImageSource**
```c#
public class FontImageSource : ImageSource
{
    // ...
    public static double DefaultSize = 30d;
    // ...
}
```

**FontImageExtension**
```c#
public class FontImageExtension : IMarkupExtension<ImageSource>
{
    // ...
    public double Size { get; set; } = FontImageSource.DefaultSize;
    // ...
}
```

Reply:
You can access it directly off of the SizeProperty

```csharp
public double Size { get; set; } = (double)FontImageSource.SizeProperty.DefaultValue;
```

Reply:
That is a great suggestion, I'll go an update that to use the `FontImageSource.SizeProperty.DefaultValue`